### PR TITLE
feat(Livestream): Add samples and tests

### PR DESCRIPTION
**Pull request by irataxy**
b:241866780

Part 1 of 3: Inputs. See also https://cloud.google.com/livestream/docs/how-to/create-input-endpoints

### CODE REVIEW

File: testutil/TestUtil.cs [hunk 1]
```diff
@@ -77,9 +77,9 @@ public T Eventually<T>(Func<T> func)
                 catch (Exception e) when (ShouldCatch(e) && i < MaxTryCount)
                 {
                     int jitteredDelayMs;
-                    lock(_lock)
+                    lock (_lock)
```
Comment (amanda-tarafa) [inline testutil/TestUtil.cs:80]: Please revert these changes, although they are good changes, it's best to keep this PR clean of unrelated changes. Feel free to push a new PR with these.

File: media/livestream/api/LiveStream.Samples/GetInput.cs [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+// [START livestream_get_input]
+
+using Google.Cloud.Video.LiveStream.V1;
+
+public class GetInputSample
+{
+    public Input GetInput(
+         string projectId, string locationId, string inputId)
+    {
+
```
Comment (amanda-tarafa) [inline media/livestream/api/LiveStream.Samples/GetInput.cs:26]: nit: remove blank line, I've seen it in a couple other samples as well

File: media/livestream/api/LiveStream.Samples/DeleteInputAsync.cs [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+// [START livestream_delete_input]
+
+using Google.Cloud.Video.LiveStream.V1;
+using Google.LongRunning;
+using Google.Protobuf.WellKnownTypes;
+
+public class DeleteInputSample
+{
+    public async void DeleteInputAsync(
```
Comment (amanda-tarafa) [inline media/livestream/api/LiveStream.Samples/DeleteInputAsync.cs:25]: An async method should never be void, else, you don't give the caller the opportunity to await on the result themsevels.
```suggestion
    public async Task DeleteInputAsync(
```

File: media/livestream/api/LiveStream.Samples.Tests/DeleteInputAsyncTests.cs [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+        {
+            await _createSample.CreateInputAsync(
+                    _fixture.ProjectId, _fixture.LocationId,
+                    _inputId);
+        }
+
+        [Fact]
+        public async Task DeletesInputAsync()
+        {
+            _deleteSample.DeleteInputAsync(_fixture.ProjectId, _fixture.LocationId, _inputId);
```
Comment (amanda-tarafa) [inline media/livestream/api/LiveStream.Samples.Tests/DeleteInputAsyncTests.cs:49]:
```suggestion
            await _deleteSample.DeleteInputAsync(_fixture.ProjectId, _fixture.LocationId, _inputId);
```

**Review by amanda-tarafa — CHANGES_REQUESTED** [review]
As always, looking good, thanks!
A couple of changes only.

I'll look at the tests as well and why they continue to fail.

**Comment by amanda-tarafa** [comment]
All failures were permission related. I've given the correct permissions to the account and tests are running again.

**Comment by amanda-tarafa** [comment]
That las failure was Kokoro itself failing after all tests were green. It happens from time to time. But it means that the tests are green.

File: testutil/TestUtil.cs (hunk 1 above)
Comment (irataxy) [reply testutil/TestUtil.cs]: Reverted

File: media/livestream/api/LiveStream.Samples/DeleteInputAsync.cs (hunk 3 above)
Comment (irataxy) [reply media/livestream/api/LiveStream.Samples/DeleteInputAsync.cs]: Done

File: media/livestream/api/LiveStream.Samples.Tests/DeleteInputAsyncTests.cs (hunk 4 above)
Comment (irataxy) [reply media/livestream/api/LiveStream.Samples.Tests/DeleteInputAsyncTests.cs]: Added await

File: media/livestream/api/LiveStream.Samples/GetInput.cs (hunk 2 above)
Comment (irataxy) [reply media/livestream/api/LiveStream.Samples/GetInput.cs]: Done

**Review by amanda-tarafa — APPROVED** [review]
LGTM

**Pull request merged**